.github/workflows/linux-build-base.yml

```diff
@@ -103,6 +103,7 @@ jobs:
             "-DVELOX_ENABLE_WAVE=ON"
             "-DVELOX_MONO_LIBRARY=ON"
             "-DVELOX_BUILD_SHARED=ON"
+            "-DVELOX_ENABLE_LOAD_SIMD_VALUE_BUFFER=ON"
```

Comment:
Do we need this change? If we can only have one run of CI, then, perhaps, we want to test non-legacy code, no? If we want to test legacy code, we'd need to add another CI run. Not sure it is worth it though.

CC: @kgpai